### DISCUSSION TITLE
Need Apache 2.0 repository license

### DISCUSSION THREAD

**Discussion by sbilello** [graphql/graphql-spec#1117] [General]
Hi team,

To contribute code from my corporate laptop, I need an Apache 2.0 License to be added to this repository. This is a requirement for open-source contributions from my company.

Would it be possible to add the license file to the repository?

Thanks in advance for your time!

**Comment by benjie** [comment]
Contributions made under the Apache 2.0 license would not be compatible with the MIT license, so I don't think we can do that unfortunately. You'll need to negotiate an exception to your contract terms with your company.
